acceptance-tests/src/test/java/it/com/atlassian/bitbucket/jenkins/internal/SmokeTest.java

```diff
@@ -725,7 +725,7 @@ private void verifySuccessfulBuild(Build lastBuild) {
                 .withMessage("Timed out waiting for the build to start")
                 .withTimeout(ofMinutes(BUILD_START_TIMEOUT_MINUTES))
                 .until(Build::hasStarted);
-        assertThat(lastBuild.getResult(), is(SUCCESS.name()));
+        gassertThat(lastBuild.getResult(), is(SUCCESS.name()));
```

Comment:
? what is `gassertThat` ? a typo perhaps?

Reply:
🤦🏼 arg yep, was typing 'g' to escape in emacs.